migrations/000020_permanent_id_request_types.up.sql

```diff
@@ -0,0 +1,19 @@
+BEGIN;
+
+SET search_path = public, pg_catalog;
+
+--
+-- The possible Permanent ID types of each request.
+--
+CREATE TABLE IF NOT EXISTS permanent_id_request_types (
```

Comment:
Should this table's initial data (now in `old/main/data/02_permanent_id_request_types.sql`) be added by this migration?
https://github.com/cyverse-de/metadata-db/blob/9907bd989e0d0328ac22f565a7981a8ae99e08b1/src/main/data/02_permanent_id_request_types.sql#L1-L5